### DISCUSSION TITLE
Get partition from run instance to trigger runrequest for sensor

### DISCUSSION THREAD

**Discussion by Sarbadi** [dagster-io/dagster#17113] [Q&A] [answered]
I have a an asset that has daily partitions attached to it. This asset is triggered by a sensor every time a specific file appears in our storage. once the sensor has run successfully I would like to trigger another sensor which with another asset that also has a partition attached to it. 
the sensor for this partition looks like this:

```python
@sensor(job=job2)
def job2_sensor(context):

    #list of run records
    run_records = context.instance.get_run_records(
        filters = RunsFilter(
        job_name = "job1",
        statuses = [DagsterRunStatus.SUCCESS]
        )
    )
    
    for run_record in run_records:
        yield RunRequest(
            run_key = run_record.dagster_run.run_id,
            partition_key = run_record.dagster_run. // I would like to pass the partition here. but I'm lost on how to do this.
        )
```

I need to run the same partition as job1, so that job2 also has the same partition.

**Comment by sryza** [comment, marked as answer]
Hi @Sarbadi - you can use `run_record.dagster_run.tags["dagster/partition"]` to get the partition from the `DagsterRun`
